build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -234,15 +241,8 @@ private void GenerateRuntimeGraph(DotNetCli dotnetCli, string destinationDeps)
             }
         }
 
-        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot, string hostFxrVersion)
+        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot)
         {
-            File.Copy(
-                Path.Combine(_corehostLockedDirectory, HostArtifactNames.DotnetHostBaseName),
```

Comment:
@gkhanna79 is this a valid scenario ? if so then we will have to look for a different way to block standalone activation from dotnet.exe

@eerhardt then what was the point in adding the following in runtimeconfig.json ? 
``` 
"framework": {
      "name": "Microsoft.NETCore.App",
      "version": "2.0.0-beta-001791-00"
    }
```